final_task/rss_parser/Handler.py

```diff
@@ -0,0 +1,60 @@
+import json
+from dataclasses import dataclass
+import feedparser
+
+from rss_parser.Entry import Entry
+
+
+@dataclass()
```

Comment:
It's great, that you have used dataclasses, but they do not suppose to contain logic. In this particular case, class Hadler should be a default class